eks/cluster-loader/clusterloader2/addon.go

```diff
@@ -93,10 +93,19 @@ func (c *ClusterLoader) buildConfigMapData() (map[string]string, error) {
 		bodyString := strings.ReplaceAll(string(body), "\n", "\n    ")
 		dataMap[path.Base(uri)] = bodyString
 	}
+	if len(c.Config.Spec.ClusterLoader.TestOverrides) > 0 {
+		overrides := strings.Join(c.Config.Spec.ClusterLoader.TestOverrides, "\n    ")
```

Comment:
You're using this indentation in multiple places. Can you extract out a constant IndentedNewline = "\n    "

Reply:
Good idea